Assets/Scripts/BeginExperiment.cs

```diff
@@ -19,18 +19,20 @@ public class BeginExperiment : MonoBehaviour
     public UnityEngine.UI.Toggle useElememToggle;
 
     // TODO: JPB: Make these configuration variables
-    private const bool HOSPITAL_COURIER = true;
+    private const bool EFR_COURIER = true;
     private const bool NICLS_COURIER = false;
     private const bool VALUE_COURIER = false;
 
-    string experiment_name = HOSPITAL_COURIER ? "StandardCourier" :
+
+    string experiment_name = EFR_COURIER ? "EFRCourier" :
                              NICLS_COURIER ? "NiclsCourier" :
-                             "StandardCourier";
+                             VALUE_COURIER ? "ValueCourier" :
+                             "Courier";
 
     private const string scene_name = "MainGame";
 
     public const string EXP_NAME_COURIER = "Courier";
-    public const string EXP_NAME_HOSPITAL = "StandardCourier";
+    public const string EXP_NAME_EFR = "EFR" + EXP_NAME_COURIER;
```

Comment:
Yeah actually, ask me about this. I was wrong, but there is something to improve here.